2nd-gen/packages/swc/package.json

```diff
@@ -55,8 +55,8 @@
         "lit": "^2.5.0 || ^3.1.3"
     },
     "devDependencies": {
-        "@adobe/postcss-token": "0.0.1",
-        "@adobe/swc-tokens": "0.0.1",
+        "@adobe/postcss-token": "workspace:*",
+        "@adobe/swc-tokens": "workspace:*",
```

Comment:
Makes versions workspace relative to assist in keeping changes in sync

TBD: an alternate method if we eventually decide to publish these packages for consumers